edx-platform/pearson-vue-theme/lms/templates/header/header.html

```diff
@@ -42,7 +52,7 @@
   </script>
 % endif
 
-<header class="global-header ${'slim' if course else ''}">
+<header class="global-header ${'slim' if course else ''} vue-header">
```

Comment:
Where are we using this class? @sergivalero20

Reply:
@Squirrel18  Only it's used to add an style in _navigation.scss (Juniper Version)

Reply:
[style](https://github.com/Pearson-Advance/openedx-themes/commit/181ab602d532a2a20d64dec68cd712205dbb4ba2#diff-b85f8ec1c03827fdbd450944cf93e466e7e58a438b61f9131ad4e9bf2e1844e6R14)